commit cb224849bc10b8cc0c60fe54b5bf2287296e33c0
Author: Samuel Thibault <samuel.thibault@ens-lyon.org>
Date:   Sun Apr 21 11:39:19 2019 +0200

    acpi: Fix static link
    
    * acpi/Makefile (HURDLIBS): Add iohelp ihash.

diff --git a/acpi/Makefile b/acpi/Makefile
index b3b7407b..f84f4b35 100644
--- a/acpi/Makefile
+++ b/acpi/Makefile
@@ -25,7 +25,7 @@ SRCS           = main.c netfs_impl.c acpi.c \
 MIGSRCS        =
 OBJS           = $(patsubst %.S,%.o,$(patsubst %.c,%.o, $(SRCS) $(MIGSRCS)))
 
-HURDLIBS= fshelp ports shouldbeinlibc netfs
+HURDLIBS= fshelp ports shouldbeinlibc netfs iohelp ihash
 LDLIBS = -lpthread
 
 target = acpi
